ts/packages/cli/src/commands/export.ts

```diff
@@ -213,26 +267,67 @@ export default class Export extends BaseCommand<typeof Export> {
     return caseFunction ? _.mapKeys(result, (value, key) => caseFunction(key)) : result;
   }
 
+  filterFromFlags(): (({ context, result }: EvalCommandReturn['string']) => boolean) | undefined {
+    const filterFlags = _.pickBy(this.flags, (value, key) => Object.values(FilterFlag).includes(key as FilterFlag));
+    if (!_.isEmpty(filterFlags)) {
```

Comment:
NIT: I prefer we go for the early return pattern and do:
```node.js
 if (_.isEmpty(filterFlags)) {
    return undefined;
  }
```

This will make the code more readable and reduce indenting